Frameworks/QuartzCore/CALayer.mm

```diff
@@ -538,15 +538,7 @@ - (void)display {
                 CreateLayerContentsBitmapContext32(width, height, priv->contentsScale)) };
             _CGContextPushBeginDraw(drawContext);
 
-            if (priv->_backgroundColor == nil || (int)[static_cast<UIColor*>(priv->_backgroundColor) _type] == solidBrush) {
-                CGContextClearToColor(drawContext,
-                                      priv->backgroundColor.r,
-                                      priv->backgroundColor.g,
-                                      priv->backgroundColor.b,
-                                      priv->backgroundColor.a);
-            } else {
-                CGContextClearToColor(drawContext, 0, 0, 0, 0);
-
+            if (priv->_backgroundColor != nil && (int)[static_cast<UIColor*>(priv->_backgroundColor) _type] != solidBrush) {
```

Comment:
@msftfox this is straight from your pull request. however, i'm concerned about what it's checking.

It looks like it will NOT fill the background if there *is* a solid color/solid brush background color?